### PR TITLE
add function for custom rank icons displayed below nametags

### PR DESCRIPTION
**Pull request by commy2**
**When merged this pull request will:**
- Adds `ace_nametags_fnc_setFactionRankIcons` which can be used to set custom rank icons displayed below name tags

### CODE REVIEW

**Comment by esteldunedain** [comment]
Perfect!

File: addons/nametags/functions/fnc_drawNameTagIcon.sqf [hunk 1]
```diff
@@ -37,7 +37,12 @@ _fnc_parameters = {
         _size = 1;
     } else {
         if (_drawRank && {rank _target != ""}) then {
-            _icon = format["\A3\Ui_f\data\GUI\Cfg\Ranks\%1_gs.paa", toLower rank _target];
+            _icon = GVAR(factionRanks) getVariable faction _target;
+            if (!isNil "_icon") then {
+                _icon = _icon param [ALL_RANKS find rank _target, ""];
```
Comment (esteldunedain) [inline addons/nametags/functions/fnc_drawNameTagIcon.sqf]: Hold on, where does ALL_RANKS come from?

**Comment by Skengman2** [comment]
Here are some Russian rank icons that I made. Feel free to use for this:
[russian_rank_icons_ace3.zip](https://github.com/acemod/ACE3/files/333270/russian_rank_icons_ace3.zip)

**Comment by commy2** [comment]
ref: #3995

**Comment by Glowbal** [comment]
#4004

**Pull request closed without merging**